### PR TITLE
CASSANDRASC-100 Fix flaky SSTableImportHandlerTest

### PR DESCRIPTION
**Pull request by sarankk**

### CODE REVIEW

File: src/test/java/org/apache/cassandra/sidecar/routes/sstableuploads/BaseUploadsHandlerTest.java [hunk 1]
```diff
@@ -222,4 +229,53 @@ public SidecarConfiguration configuration()
             return sidecarConfiguration;
         }
     }
+
+    static class TestCassandraAdapterDelegate extends CassandraAdapterDelegate
```
Comment (frankgh) [inline src/test/java/org/apache/cassandra/sidecar/routes/sstableuploads/BaseUploadsHandlerTest.java:233]: can we call this `FakeCassandraAdapterDelegate` instead?

Comment (sarankk) [reply src/test/java/org/apache/cassandra/sidecar/routes/sstableuploads/BaseUploadsHandlerTest.java:233]: But we already follow a pattern of calling such dummy classes Test<>. Example https://github.com/apache/cassandra-sidecar/blob/c72f2179143e7e031f247d3e8385a29c5e64c1c3/src/test/java/org/apache/cassandra/sidecar/routes/restore/BaseRestoreJobTests.java#L168

**Review by frankgh — APPROVED** [review]
+1 looks good to me

**Comment by frankgh** [comment]
Closed via https://github.com/apache/cassandra-sidecar/commit/2914d57f0428643b3a92b6af8f4da1b209d80c2a

**Pull request closed without merging**